### ISSUE TITLE
A11Y: aira-lable could not be inserted to InteriorLeftNavList

### ISSUE THREAD

**Issue by bjliuyi** [carbon-design-system/carbon-components-react#505]
## Detailed description
Followed with accessibility checkpoint, WAI-ARIA widgets must have an accessible name specified with aria-label, aria-labelledby or the widget's inner text. But, InteriorLeftNavList is failed in this point.

Describe in detail the issue you're having. Is this a feature request (new component, new icon), a bug, or a general issue?
it is a general issue.

> Is this issue related to a specific component?
No

> What did you expect to happen? What happened instead? What would you like to see changed?
WAI-ARIA widgets have aria-label.

> What browser are you working in?
Chrome

> What version of the Carbon Design System are you using?
"carbon-components-react": "5.0.6",

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/34873342/35083757-a7143dde-fc5b-11e7-92df-9c72a8b6bf66.png)

## Add labels
aria-lable
